src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,27 @@
+package calculator;
+
+import calculator.element.Element;
+import calculator.element.operand.Operand;
+import calculator.element.operaor.*;
+import calculator.deque.ElementDequeFactory;
+import calculator.deque.ElementDeque;
+
+public class Calculator {
+
+    static int equality(String input) {
+        ElementDeque elementDeque = ElementDequeFactory.createElementDeque(input);
+        return operateMultiple(elementDeque);
+    }
+
+    private static int operateMultiple(ElementDeque elementDeque) {
+        while(! elementDeque.isQuitCalculate()) {
+            Operand operandA = elementDeque.popOperand();
+            Operator operator = elementDeque.popOperator();
+            Operand operandB = elementDeque.popOperand();
+            Element result = operator.operate(operandA, operandB);
+            elementDeque.addResult(result);
+        }
+
+        return elementDeque.popOperand().getValue();
```

Comment:
```suggestion
        return elementDeque.getValue(); // 또는 결과 반환을 뜻하는 적절한 메서드명
```
popOperand 해서 getValue 해야 계산 결과를 가져올 수 있다는 걸 알려면 ElementDeque 코드 내부를 잘 알아야 합니다
ElementDeque에서 따로 기능을 제공해주면 좀더 빠르게 파악할 수 있지 않을까요?

Reply:
아 체이닝일경우 단계를 줄여야되는 코드 스멜이 있군요. 감사합니다.